<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="input">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Programme Regular</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="question_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>851</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Programme Regular</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="result_label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Programme Light</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="score_label">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>530</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Programme Regular</family>
     <pointsize>25</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="endButton">
   <property name="geometry">
    <rect>
     <x>862</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Programme Bold</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Fin </string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../assets/fin.png</normaloff>../assets/fin.png</iconset>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>500</width>
     <height>10</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Programme</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
